code/workspaces/web-app/src/containers/projects/ProjectsContainer.js

```diff
@@ -28,11 +26,19 @@ class ProjectsContainer extends Component {
       key: project.key,
       displayName: project.name,
       description: project.description,
+      accessible: project.accessible,
       type: 'project',
       status: 'ready',
     }));
   }
 
+  projectUserPermissions(project) {
+    if (project === undefined) {
```

Comment:
Is it possible to enter this code block with an undefined project? No point having the check if it isn't is there?